components/tree-select/nz-tree-select.component.ts

```diff
@@ -140,7 +140,7 @@ export class NzTreeSelectComponent extends NzTreeBase implements ControlValueAcc
   isNotFound = false;
   inputValue = '';
   dropDownPosition: 'top' | 'center' | 'bottom' = 'bottom';
-  selectionChangeSubscription: Subscription;
+  selectionChangeSubscription: Subscription = Subscription.EMPTY;
```

Comment:
```suggestion
private destroy$ = new Subject<void>();
```

在 `subscribeSelectionChange` 方法中订阅之前添加 `pipe(takeUntil(this.destroy$))`

在 `ngOnDestroy` 添加 
```ts
  this.destroy$.next();
  this.destroy$.complete();
```